src/app/pages/forms-and-validation/password/password.component.html

```diff
@@ -7,14 +7,6 @@
 
 <section class="change-password-section">
     <mat-card class="change-password-card">
-        <mat-card-header>
```

Comment:
![image](https://user-images.githubusercontent.com/6538289/134372615-4700aaa9-f211-4ba6-8877-de8a61cafaa6.png)

Need to keep the title on non-mobile.  Maybe just put a `*ngIf="!isSmall"` on it.